test/test_helper/common.bash

```diff
@@ -87,7 +87,7 @@ function wait_for_smtp_port_in_container() {
 
 # @param ${1} name of the postfix container
 function wait_for_smtp_port_in_container_to_respond() {
-  until [[ $(docker exec "${1}" "echo QUIT | nc localhost 25") == *"221 2.0.0 Bye"* ]]; do
+  until [[ $(docker exec "${1}" /bin/sh -c "echo QUIT | nc localhost 25") == *"221 2.0.0 Bye"* ]]; do
```

Comment:
`until [[ $(docker exec "${1}" timeout 10 /bin/sh -c "echo QUIT | nc localhost 25") == *"221 2.0.0 Bye"* ]]; do`

If, for whatever reason, nc "blocks" (port is open, but no response from the service), abort after 10 seconds. Otherwise, netcat will just "hang".

Reply:
Yep, I wanted to be sure it worked first. I'll add the count of retries and fail now

Reply:
I think you missed the "timeout" command in my example?

Reply:
Oh yes 👍🏼